pom.xml

```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>tomtom-vector</artifactId>
-    <version>4.44.8-SNAPSHOT</version>
+    <version>5.20.4-SNAPSHOT</version>
     <name>TomTom Maps SDK</name>
     <description>TomTom Maps SDK for Web with vector maps</description>
     <url>https://developer.tomtom.com/maps-sdk-web/downloads</url>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <upstream.url>https://api.tomtom.com/maps-sdk-js/${version.unsnapshot}/jssdk-${version.unsnapshot}-distribution-vector.zip</upstream.url>
+        <upstream.url>https://api.tomtom.com/maps-sdk-for-web/5.x/${version.unsnapshot}/jssdk-${version.unsnapshot}-distribution-maps.zip</upstream.url>
```

Comment:
Is this still the same `tomtom-vector` artifact?

Reply:
Yes it's still the same SDK but with a new major version. Jar content is pretty much the same (same structure, same assets...), with the main package name changed and some new methods.